packages/blaze-html-templates/package.js

```diff
@@ -0,0 +1,21 @@
+Package.describe({
+  name: 'blaze-html-templates',
+  version: '1.0.0',
+  // Brief, one-line summary of the package.
+  summary: 'Compile HTML templates into reactive UI with Meteor Blaze',
+  // By default, Meteor will default to using README.md for documentation.
+  // To avoid submitting documentation, set this field to null.
+  documentation: 'README.md'
+});
+
+Package.onUse(function(api) {
+  api.imply([
+    // A library for reactive user interfaces
+    'blaze',
+    'ui', // XXX COMPAT WITH PACKAGES BUILT FOR 0.9.0.
+    'spacebars', // XXX COMPAT WITH PACKAGES BUILT FOR 0.9.0
```

Comment:
We should clarify all these `XXX COMPAT` messages, because these aren't packages full of back-compat code, and actually some of the reasons for including them are subtle.

For `ui`, it sounds like it's so that `Package.ui` exists.  There's literally no other effect of implying it here, which is quite subtle indeed.  For `spacebars`, it's to export the `Spacebars` symbol.  We should also comment that `ui`, `livedata`, and `deps` are empty packages.